src/components/modal/base/modal.jsx

```diff
@@ -23,10 +23,15 @@ class Modal extends React.Component {
         return this.modal.portal.requestClose();
     }
     render () {
+        // only set bodyOpenClassName prop if this.props.useStandardSizes is true.
+        // (careful -- setting bodyOpenClassName to null here causes an error, because
+        // ReactModal does not correctly handle the case of a null bodyOpenClassName)
+        const bodyOpenClassNameProp = this.props.useStandardSizes ?
+            {bodyOpenClassName: classNames('overflow-hidden')} : {};
         return (
             <ReactModal
                 appElement={document.getElementById('app')}
-                bodyOpenClassName={this.props.useStandardSizes ? classNames('overflow-hidden') : null}
+                {...bodyOpenClassNameProp}
```

Comment:
Optional classes based on the truthiness of props is a good job for `classNames`:
```jsx
                bodyOpenClassName={classNames({
                    'overflow-hidden': this.props.useStandardSizes
               })}
```

Reply:
What will the value of the prop be if it omits overflow-hidden? An empty string? That might work. I only tested null (throws an error when switching between project page and editor) vs. undefined (no error).

Reply:
I would have to check, but I'm pretty sure it's an empty string since class names are supposed to be strings.